azkaban-common/src/main/java/azkaban/jobExecutor/ProcessJob.java

```diff
@@ -58,10 +58,6 @@
   public ProcessJob(final String jobId, final Props sysProps,
       final Props jobProps, final Logger log) {
     super(jobId, sysProps, jobProps, log);
-
-    // this is in line with what other job types (hadoopJava, spark, pig, hive)
-    // is doing
-    jobProps.put(CommonJobProperties.JOB_ID, jobId);
```

Comment:
quick question, isn't ProcessJob used by all job types? Then why is JOB_ID not found?